CONTRIBUTING.md

```diff
@@ -1,8 +1,20 @@
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Getting Started                       | Presto's [open issues are here](https://github.com/prestodb/presto/issues). Tag issues that would make a good first pull request for new contributors with [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tag. An easy way to start helping the project is to *file an issue*. Issues can include bugs, new features, or outdated documentation. |
+| Building Presto                       | Please review the minimum system specs [requirements](#requirements) before [building](#buildingpresto) Presto.                                                                                                                                                                                                                                                                    |
+| Report Bug                            | To report a bug, visit Presto's [open issues](https://github.com/prestodb/presto/issues).                                                                                                                                                                                                                                                                                          |
+| Contributions                         | Please read the [contributions](#contributions) section to learn about how you can contribute to Presto, including the submission process, minimum expectations, and guidelines for designing your code.                                                                                                                               |
+| Pull Requests                         | Ready to commit your code to the main branch? Be sure to review the [Pull Request guidelines](#pullrequests) .                                                                                                                                                                                                                                                                     |
+| Contributor License Agreement ("CLA") | For first-time contributors, a CLA must be signed. Read more about it [here](#cla).                                                                                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| Contributor License Agreement ("CLA") | First-time contributors must sign a CLA. For more information see [Contributor License Agreement ("CLA")](#cla).                                                                                                                                                                                                                                                                                                |
```
Avoid "here" in links. For context, see the GitLab Documentation Style Guide entry [Descriptive text rather than here](https://docs.gitlab.com/ee/development/documentation/styleguide/#descriptive-text-rather-than-here).